server/index.ts

```diff
@@ -104,91 +104,82 @@ app.get('/api/oauth/return', async (request, response, next) => {
   }
 });
 
-app.get('/api/search/artist/:searchq', async (request, response, next) => {
-  const searchQuery = request.params.searchq;
+app.get(
+  '/api/search/:searchcategory/:searchq',
+  async (request, response, next) => {
+    const searchQuery = request.params.searchq;
+    const searchCategory = request.params.searchcategory;
+    let format;
+    {
+      searchCategory === 'title' ? (format = '&format=lp') : (format = null);
+    }
 
-  try {
-    const authCookie = JSON.parse(request.cookies.auth);
+    try {
+      const authCookie = JSON.parse(request.cookies.auth);
 
-    const token = authCookie.token;
-    const secret = authCookie.secret;
+      const token = authCookie.token;
+      const secret = authCookie.secret;
 
-    const searchResponse = await fetch(
-      `https://api.discogs.com/database/search?type=release&county=germany&artist=${searchQuery}`,
-      {
-        headers: {
-          'Content-Type': 'application/x-www-form-urlencoded',
-          Authorization: `OAuth oauth_consumer_key="${consumerKey}", oauth_nonce="${Date.now()}", oauth_token="${token}", oauth_signature="${consumerSecret}&${secret}",oauth_signature_method="PLAINTEXT",oauth_timestamp="${Date.now()}"`,
-        },
-      }
-    );
-    const search = await searchResponse.json();
-    const results = search.results;
-    const searchResult = results.map((result: SearchResultProps) => ({
-      title: result.title,
-      id: result.id,
-      cover: result.cover_image,
-      in_collection: result.user_data.in_collection,
-    }));
-    response.send(searchResult);
-  } catch (error) {
-    next(response.status(500).send('Internal Server Error'));
+      const searchResponse = await fetch(
+        `https://api.discogs.com/database/search?type=release&county=germany${format}&${searchCategory}=${searchQuery}`,
+        {
+          headers: {
+            'Content-Type': 'application/x-www-form-urlencoded',
+            Authorization: `OAuth oauth_consumer_key="${consumerKey}", oauth_nonce="${Date.now()}", oauth_token="${token}", oauth_signature="${consumerSecret}&${secret}",oauth_signature_method="PLAINTEXT",oauth_timestamp="${Date.now()}"`,
+          },
+        }
+      );
+      const search = await searchResponse.json();
+      const results = search.results;
+      const searchResult = results.map((result: SearchResultProps) => ({
+        title: result.title,
+        id: result.id,
+        cover: result.cover_image,
+        in_collection: result.user_data.in_collection,
+      }));
+      response.send(searchResult);
+    } catch (error) {
+      next(response.status(500).send('Internal Server Error'));
+    }
```

Comment:
This looks fine, but I feel compelled to ask why is everything green? 😂
In this entire file so far there has only been one change, and that was in line 124. 
Did you like, cut everything out, and paste it again? Or is that a GIt error? I heard things like that happen sometimes, but haven't seen it before 🧐